src/app/features/file/edit-photo/edit-photo.component.scss

```diff
@@ -23,3 +23,6 @@
   margin: auto;
   color: rgba(0, 0, 0, 0.6);
 }
+.img-label.invalid {
+  color: rgba(244, 67, 54, 1);
```

Comment:
How did you select this color? Looks like it matches the color of other error fields (which is great :+1: ).
Is this matching one of our general colors here (maybe $error?): https://github.com/Aam-Digital/ndb-core/blob/76f46603066192671ca869cd9e13d1355adbeb26/src/styles/variables/_colors.scss#L10

Reply:
I inspected the existing fields for error colors and used the same color for this field. 
I've updated this label to use the $error variable color for consistency. Thanks for pointing it out!